Composer/packages/ui-plugins/teams/src/actions.ts

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { UISchema } from '@bfc/extension-client';
+import { SDKKinds } from '@bfc/shared';
+import formatMessage from 'format-message';
+
```

Comment:
See the comment below (on triggers.ts) about defining constants for common values.